api/proto/teleport/legacy/types/types.proto

```diff
@@ -3247,6 +3247,19 @@ message RoleConditions {
   reserved "SAMLIdPServiceProviderLabels";
   reserved 41; // removed saml_idp_service_provider_labels_expression in favor of using app_labels_expression.
   reserved "SAMLIdPServiceProviderLabelsExpression";
+
+  // AccountAssignments holds the list of account assignments affected by this
+  // condition
+  repeated IdentityCenterAccountAssignment AccountAssignments = 42 [
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag) = "account_assignments,omitempty"
+  ];
+}
+
+// IdentityCenterAccountAssignment ...
+message IdentityCenterAccountAssignment {
+  string PermissionSet = 1 [(gogoproto.jsontag) = "permission_set,omitempty"];
+  string AccountID = 2 [(gogoproto.jsontag) = "account,omitempty"];
```

Comment:
I don't know if we have particular rules on naming the JSON fields, but I would expect this to be `account_id`.